### DISCUSSION TITLE
Suggested prefix configuration for database.php

### DISCUSSION THREAD

**Discussion by jevantang** [laravel/framework#50721] [Ideas]
`'prefix' => env('DB_PREFIX', '')`

The new framework allows products to reduce configuration files, but prefix does not yet support .env configurations. prefix is a very commonly used configuration and is recommended to be added to the framework.

**Comment by macropay-solutions** [comment]
This config already exists since laravel 5.x. Or you mean you have to edit the config to add the value from the env?

**Reply by jevantang** [reply]
Yes, the database.php file is required if the product is configured with a prefix. This is because .env configuration is not supported.

**Comment by shaedrich** [comment]
Ideally, every config value should have a corresponding (it can be optional, of course) env value.
